### PR TITLE
Get sector from CSV upload or API request

### PR DESCRIPTION
**Pull request by maurizi**
## Overview

This PR adds a new required field, `sector`, to `FacilityListItem`, and supports setting it via either CSV upload or the API

Connects #1830 

## Demo

Not a very exciting demo because Sector doesn't show up on the front-end yet much.

It does show up in some error messages if you omit `sector` from your CSV:
image

Or if it's missing from a row in the CSV:
image

# Notes

Templates with the added sector column can be found here: [OAR_Contributor_Template_CSV_Additional_Data_Points.zip](https://github.com/open-apparel-registry/open-apparel-registry/files/8781742/OAR_Contributor_Template_CSV_Additional_Data_Points.zip)

## Testing Instructions

* `scripts/resetdb`
* `scripts/server`
* Use the Contribute page to upload a CSV. Without a `sector` field you should receive an error. Add a sector column to your CSV, fill it with a value (e.g. `Apparel`) and upload it again - it should succeed
* The `makefixtures` and `geocode_csvs` management commands work correctly, but now including a `sector` column in their input/output

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines
- [x] This PR is targeted at the correct branch (`develop` vs. `ogr/develop`)
- ~[ ] If this PR applies to both OAR and OGR a companion PR has been created~

### CODE REVIEW

**Review by jwalgran — COMMENTED** [review]
All my tests were successful. In addition to some minor inline comments we just need to fix up a Swagger documentation string

https://github.com/open-apparel-registry/open-apparel-registry/blob/01caeeb61292f83dad3c93860a24ca6e00dbab3b/src/django/api/views.py#L880-L882

This now needs to be something like
"The country, name, address, and sector(s) of the facility. "

File: src/django/api/tests.py [hunk 1]
```diff
@@ -401,12 +404,14 @@ def test_raises_if_item_is_not_uploaded(self):
 
     def test_parses_using_header(self):
         facility_list = FacilityList.objects.create(
-            header='address,country,name')
+            header="sector,address,country,name"
```
[the remark this answers is not in the document]
Comment (maurizi) [reply src/django/api/tests.py:407]: Yeah - I accidentally still had `black` enabled and it formatted this file. I saw that the quoting is _mostly_ single-quotes but not completely consistent within the project, so I didn't try and revert everything it did, just the unrelated changes.

File: src/django/api/models.py [hunk 2]
```diff
@@ -636,6 +636,10 @@ class Meta:
         help_text=('The ISO 3166-1 alpha-2 country code of the facility taken '
                    'directly from the raw data or looked up based on a full '
                    'country name provided in the raw data.'))
+    sector = postgres.ArrayField(
+        models.CharField(max_length=200, null=False, blank=False),
```
[the remark this answers is not in the document]
Comment (maurizi) [reply src/django/api/models.py]: Ah woops - I initially had it at `200` and thought that might be too big so I changed to to `50`, but I guess the changes to `models.py` didn't get committed. I can make this consistent at whatever number seems appropriate.

[the remark this answers is not in the document]
Comment (maurizi) [reply src/django/api/models.py]: Added a fixup commit to fix this

File: src/django/api/views.py [hunk 3]
```diff
@@ -878,8 +878,8 @@ def get_serializer_fields(self, path, method):
                     name='data',
                     location='body',
                     description=(
-                        'The country, name, and address of the facility. See '
-                        'the sample request body above.'),
+                        'The country, name, address and sector(s) of the '
+                        'facility. See the sample request body above.'),
```
Comment (maurizi) [inline src/django/api/views.py:882]: @jwalgran this accidentally got rebased into this commit before the review was done - just wanted to flag that this did get updated

**Pull request merged**